boards/xtensa/esp32/common/src/esp32_board_spi.c

```diff
@@ -71,15 +71,17 @@ static inline uint8_t spi_status(struct spi_dev_s *dev, uint32_t devid)
 static inline int spi_cmddata(struct spi_dev_s *dev, uint32_t devid,
                               bool cmd)
 {
-#ifdef CONFIG_LCD_ILI9341
+#if defined(CONFIG_LCD_ILI9341) || defined(CONFIG_LCD_SSD1680)
   if (devid == SPIDEV_DISPLAY(0))
     {
       /*  This is the Data/Command control pad which determines whether the
        *  data bits are data or a command.
        */
-
+#  if defined(CONFIG_LCD_SSD1680)
+      esp32_gpiowrite(CONFIG_SSD1680_GPIO_PIN_DTA_CMD, !cmd);
+#  else
       esp32_gpiowrite(DISPLAY_DC, !cmd);
-
+#  endif
```

Comment:
I think the current implementation of `CONFIG_SSD1680_GPIO_PIN_DTA_CMD` could be optimized. We need to change `board.h`:
```
#define DISPLAY_SPI       2
#if defined(CONFIG_LCD_SSD1680)
#  define DISPLAY_DC CONFIG_SSD1680_GPIO_PIN_DTA_CMD
#else
#  define DISPLAY_DC        21
#endif
#define DISPLAY_RST       18
#define DISPLAY_BCKL      5
```
and revert change here